catroid/src/arduino/res/xml/preferences.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this poses the problem that if we add another feature, someone has to remember that preferences have now be modified here and in the main package.

maybe moving the default values of preferences somewhere else like the flavoredConstants or creating another array xml to store those values.